### PR TITLE
Update identifying attribute for jdbc backend

### PR DESCRIPTION
**Pull request by rish691**

### CODE REVIEW

File: hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/JdbcBackendResolver.java [hunk 1]
```diff
@@ -77,7 +77,7 @@
           EntityConstants.getValue(BackendAttribute.BACKEND_ATTRIBUTE_PATH),
           createAttributeValue(path));
     }
-    entityBuilder.putIdentifyingAttributes(RawSpanConstants.getValue(Sql.SQL_DB_TYPE),
+    entityBuilder.putAttributes(RawSpanConstants.getValue(Sql.SQL_DB_TYPE),
```
Comment (tim-mwangi) [inline hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/JdbcBackendResolver.java:80]: Didn't this burn us a while ago in prod?

Comment (tim-mwangi) [reply hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/JdbcBackendResolver.java:80]: CC @buchi-busireddy

Comment (rish691) [reply hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/JdbcBackendResolver.java:80]: What do you mean?

Comment (buchi-busireddy) [reply hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/JdbcBackendResolver.java:80]: If this isn't an identifying attribute for BACKEND entity, it should be fine.

```
rs0:PRIMARY> db.entity_types.find({name:"BACKEND"}).pretty()
{
	"_id" : "__root:BACKEND",
	"_lastUpdateTime" : ISODate("2020-08-06T17:36:06.202Z"),
	"attributeType" : [
		{
			"name" : "BACKEND_HOST",
			"valueKind" : "TYPE_STRING",
			"identifyingAttribute" : true
		},
		{
			"name" : "BACKEND_PORT",
			"valueKind" : "TYPE_STRING",
			"identifyingAttribute" : true
		},
		{
			"name" : "BACKEND_PATH",
			"valueKind" : "TYPE_STRING"
		},
		{
			"name" : "BACKEND_PROTOCOL",
			"valueKind" : "TYPE_STRING",
			"identifyingAttribute" : true
		}
	],
	"createdTime" : NumberLong("1596656818940"),
	"name" : "BACKEND",
	"tenantId" : "__root"
}
```

**Comment by buchi-busireddy** [comment]
@rish691 i'm just curious about what kind of issues this was causing?

**Review by tim-mwangi — APPROVED** [review]

**Comment by rish691** [comment]
> @rish691 i'm just curious about what kind of issues this was causing?

This causes exception while registering backend entity. 
`java.lang.IllegalArgumentException:` Received and expected identifying attributes differ. Received: [BACKEND_PROTOCOL, BACKEND_HOST, BACKEND_PORT, sql.db_type] . Expected: [BACKEND_HOST, BACKEND_PROTOCOL, BACKEND_PORT]`

**Pull request merged**